qemu/tests/verify_panic_status_with_pvpanic.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+import aexpect
+
+
+def run(test, params, env):
+    """
+    Verify the QMP even with -device pvpanic when trigger crash,this case will:
+
+    1) Start VM with pvpanic device.
+    2) Check if pvpanic device is exist in guest.
+    3) Trigger crash in guest.
+    4) Check vm status with QMP.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environmen.
+    """
+
+    qmp_command = params["qmp_command"]
+    stop_kdump_command = params["stop_kdump_command"]
+    trigger_crash = params["trigger_crash"]
+    qmp_check_info = params["qmp_check_info"]
+    check_info = params.get("check_info")
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login()
+    qmp_port = vm.monitor
+
+    if check_info:
+        qtree_info = vm.monitor.info("qtree")
+        if params.get("check_info") not in qtree_info:
+            test.fail("Not find pvpanic device in guest")
+
+    try:
+        session.cmd(stop_kdump_command)
+        session.cmd(trigger_crash, timeout=10)
+    except aexpect.ShellTimeoutError:
+        pass
+    finally:
+        output = qmp_port.get_status()
+        logging.info(output)
+        if qmp_check_info not in str(output):
+            test.fail("Guest status is not guest-panicked")
+
+    session.close()
```

Comment:
Move this line to `finally:`
```suggestion
if session:
    session.close()
```